src/help/plugin.ts

```diff
@@ -149,19 +148,21 @@ function activate(app: JupyterLab, mainMenu: IMainMenu, palette: ICommandPalette
   // Handle state restoration.
   restorer.restore(tracker, {
     command,
-    args: widget => ({ isHidden: widget.isHidden, url: widget.url }),
-    name: widget => namespace
+    args: widget => ({ url: widget.url }),
+    name: widget => widget.id
```

Comment:
I'd suggest returning `name: widget => widget.url` here because it won't have spaces in it and is unique. The `id` you're using below is the `text` attribute, which is just the title of each help page, so it has spaces.

In addition, below, I would set the `widget.id` to something that isn't necessarily deterministic, just creating a counter and:
```
widget.id = `${namespace}-${++counter}`;
```